blog/_posts/2023-03-20-curation-model.md

```diff
@@ -0,0 +1,72 @@
+
+<div class="text-center">
+    <img class="img-thumbnail" src="{{ site.baseurl }}/blog/images/ads-curation_model_f2.png" />
+<em>Figure 2: Directed network based on citation relationship between journals. The nodes represent journals. This is a fictitious network, for illustration purposes only.
+</em>
+</div>
+<br>
+
+At its core, the algorithm that calculates the Eigenfactor values is the well-known [PageRank](https://en.wikipedia.org/wiki/PageRank) algorithm, which properly deals with the fact that journals are all interconnected in a citation network (see figure 2, showing a fictitious network). The [Journal Citation Reports](https://en.wikipedia.org/wiki/Journal_Citation_Reports) (JCR), published by Clarivate Analytics, provides the Journal Eigenfactor on an annual basis. However, we prefer not to have to depend on an external data source for our decisions. That is why we made a comparison between Journal Eigenfactors calculated using ADS citation data and those listed in the JCR for a number of years.
+
+The results are shown in figure 3. The fact that values are not equal is no surprise, because the citation networks in the ADS and Web of Science are different. The encouraging result is that the values have a very strong correlation.
```

Comment:
Web of Science? I thought this was JCR data? If those are the same, make sure to say so explicitly, or just replace WoS with JCR for simplicity.